index.js

```diff
@@ -45,8 +45,6 @@ module.exports = (api, options) => {
         },
       ]);
 
-    webpackConfig.output.set("jsonpFunction", `webpackJsonp__${name}`);
```

Comment:
I'd prefer having one version of vue-cli-plugin-single-spa that works with both webpack 4 and 5, if possible. Easier to maintain.

```js
if (lessThanWebpack5()) {
  webpackConfig.output.set("jsonpFunction", `webpackJsonp__${name}`);
}

function lessThanWebpack5() {
  const semver = require('semver')
  const webpack = requre('webpack')
  return semver.satisfies(webpack.version, '<5')
}
```

Note that semver would need to be added as a dependency. Also that it might be safest to do something like the following to ensure that the correct installed instance of webpack is used wwhen checking:

```js
const webpack = require(require.resolve('webpack', {paths: [require.resolve('@vue/cli-plugin-webpack-4')]}))
```

That ensures the returned webpack module is the one going to be used by vue-cli, rather than a different instance/version of webpack. However, it's tricky because providing the `@vue/cli-plugin-webpack-4` upfront defeats the purpose of checking webpack 4/5. Might be possible to just use `@vue/cli`, but I haven't tested and not sure if that would provide the needed effect. For that reason, I didn't include it in my suggestion code above. Feel free to explore the require.resolve() approach further if you'd like, or I'm happy to merge without it.